docker/Dockerfile.test

```diff
@@ -0,0 +1,30 @@
+FROM datajoint/djbase:py3.9-debian-8eb1715
+
+# ARG GITHUB_USERNAME=datajoint # tried moving to ENV
+USER anaconda:anaconda
+
+COPY ./workflow-optogenetics/docker/apt_requirements.txt /tmp/
+RUN /entrypoint.sh echo "Installed dependencies."
+
+WORKDIR /main/workflow-optogenetics
+
+# Always get interface/djarchive
+RUN pip install --no-deps "element-interface@git+https://github.com/datajoint/element-interface"
```

Comment:
```suggestion
```

Reply:
Can we install from PyPI since this package is now published?

Reply:
Sure, I'll make that adjustment. I think I initially submitted this work prior to that publication